scripts/pi-hole/help.js

```diff
@@ -1,6 +1,6 @@
 $( "#flush" ).click(function() {
     if (confirm("Are you sure you want to flush the Pi-hole log file?")) {
-        document.location.href="help.php?flush";
+        document.location.href="../../help.php";
```

Comment:
This is a bad modification. See it got rid of the `?flush` rendering the whole function useless!